code/game/machinery/vending.dm

```diff
@@ -2426,6 +2426,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/clothing/suit/wizrobe/magician/fake = 3,
 		/obj/item/clothing/head/wizard/magician = 3,
 		/obj/item/clothing/suit/sakura_kimono = 3,
+		/obj/item/clothing/gloves/white = 3,
```

Comment:
No need for this ,

Reply:
yeah no need for it but also no need not for it

Reply:
/shrug